lib/src/mixer/mixer.dart

```diff
@@ -401,6 +401,10 @@ class Mixer {
         ),
       );
 
+      if (mixer.backgroundColor?.hasAnimation ?? false) {
+        mixer.decoration!.animated();
+      }
```

Comment:
This is not the best approach IMO, but it seems it's the only option for now. Animating the whole decoration because only one property (color) is animated is not a good thing.